.gitmodules

```
@@ -1,3 +1,3 @@
-[submodule "temporal-proto"]
-	path = temporal-proto
-	url = git@github.com:temporalio/temporal-proto.git
\ No newline at end of file
+[submodule "temporal-api"]
+	path = temporal-api
```

```suggestion
	path = proto/api
```
for consistency with other repos.